docs/appkit/migration/from-connectkit-next.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Migration from ConnectKit to AppKit
+pagination_next: 
+---
+
+# Migration from ConnectKit to AppKit
+
+If you have currently set up ConnectKit as the wallet provider for your Web3 app, you can easily migrate to **Reown AppKit**.
+
+Assuming that your Web3 app is set up similarly to ConnectKit's example app [here](https://github.com/family/connectkit/tree/main/examples/nextjs), let's explore how to migrate from this example app to Reown AppKit.
+
+To migrate from ConnectKit to Reown AppKit, please follow the steps below.
+
+## Step 1. Create a project in Reown Cloud
+
++ Create a new project on [Reown Cloud](https://cloud.reown.com) and obtain a new project ID.
+
+## Step 2. Install & uninstall libraries
+
++ Run this command to install Reown AppKit and uninstall ConnectKit.
+
+```bash npm2yarn
+npm install @reown/appkit @reown/appkit-adapter-wagmi && npm uninstall connectkit
+```
+
+## Step 3. Change the code in the `/components/Web3Provider.tsx` file
+
++ Navigate to the `/components/Web3Provider.tsx` file inside your ConnectKit example Web3 app directory. 
++ Now, you need to remove the existing configuration that uses ConnectKit and replace it with Reown AppKit. Refer to the code snippet below.
+
+```tsx
+import React from 'react';
+
+import { WagmiProvider, createConfig } from 'wagmi';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+/* highlight-delete-start */
+- import { ConnectKitProvider, getDefaultConfig } from 'connectkit';
+/* highlight-delete-end */
+/* highlight-add-start */
++ import { WagmiAdapter } from '@reown/appkit-adapter-wagmi'
+
+// Include networks from AppKit
++ import { mainnet, arbitrum } from '@reown/appkit/networks'
++ import { createAppKit } from '@reown/appkit';
++ export const networks = [mainnet, arbitrum]
+/* highlight-add-end */
+
+/* highlight-delete-start */
+- const config = createConfig(
+-   getDefaultConfig({
+-     appName: 'ConnectKit Next.js demo',
+-     walletConnectProjectId: process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID!,
+-   })
+- );
+/* highlight-delete-end */
+
+/* highlight-add-start */
+// Get projectId from https://cloud.reown.com
++ export const projectId = "YOUR_PROJECT_ID";
```

Comment:
If they are using WalletConnect QR, they should have a projectId, so maybe we can suggest to use the same.

Reply:
Added it 🫡